github/data_source_github_repositories_test.go

```diff
@@ -50,7 +76,16 @@ func TestAccGithubRepositoriesDataSource_noMatch(t *testing.T) {
 func testAccCheckGithubRepositoriesDataSourceConfig(query string) string {
 	return fmt.Sprintf(`
 data "github_repositories" "test" {
-	query = "%s"
+	query = %q
```

Comment:
Should we be removing the ""s from around string values? 🤔 While that may work in .12 some people may use the tests as an example and still be running .11. Would it be better to use the syntax that has the most compatibility until we officially retire .11?

Reply:
I just realized %q quoted 😅 It might make make more sense to leave this as "%s" to be explicit that there are quotes there.

Reply:
I originally left it like that. But found over in AWS land, we are trying to be more consistent, so we use `%q` since that's what it does.